tests/Behat/Content/Node/EventContentContext.php

```diff
@@ -91,6 +94,23 @@ public function alterEventFields(BeforeParseEntityFieldsScope $scope): void {
           $scope->addFields($fields)->removeField($key);
           break;
 
+        // Set Authors entity reference fields.
+        case 'Authors':
+          $ids = [];
+          $revision_ids = [];
+          $names = explode(', ', $value);
+          foreach ($names as $name) {
+            $entity = $this->subEntityContext->getSubEntityByName($name);
+            $ids[] = $entity->id();
+            $revision_ids[] = $entity->getRevisionId();
+          }
+          $scope->addFields([
+            $mapping[$key] . ':target_id' => implode(',', $ids),
+            $mapping[$key] . ':target_revision_id' => implode(',', $revision_ids),
+          ]);
+          $scope->removeField($key);
+          break;
```

Comment:
We are repeating this block of code too much, let's follow the example of:

```
        case 'Contacts':
          $fields = $this->getReferenceRevisionField($scope->getEntityType(), $scope->getBundle(), $mapping[$key], $value);
          $scope->addFields($fields)->removeField($key);
          break;
```
Let's:
- Create a trait called `SubEntityReferenceTrait`
- Add a method called `getSubEntityReferenceField(...)`
- Refactor all context to use that trait/method